src/pulsar_producer.erl

```diff
@@ -279,3 +431,185 @@ next_sequence_id(State = #state{sequence_id = SequenceId}) ->
     State#state{sequence_id = SequenceId+1}.
 
 log_error(Fmt, Args) -> logger:error("[pulsar-producer] " ++ Fmt, Args).
```

Comment:
need the `State` here, so we can log which partitiontopic is this error about.

Reply:
:heavy_check_mark: